compendium/DeclarativeServices/src/manager/ConfigurationManager.hpp

```diff
@@ -94,12 +96,13 @@ namespace cppmicroservices
             bool isConfigSatisfied() const noexcept;
 
             std::shared_ptr<cppmicroservices::logservice::LogService> logger; ///< logger for this runtime
-            const std::shared_ptr<const metadata::ComponentMetadata> metadata;
+            std::shared_ptr<metadata::ComponentMetadata const> const metadata;
             cppmicroservices::BundleContext bundleContext; ///< context of the bundle which contains the component
             mutable std::mutex propertiesMutex; // mutex to protect the configProperties and mergedProperties members
             std::unordered_map<std::string, cppmicroservices::AnyMap>
                 configProperties; // properties for available configuration objects.
             cppmicroservices::AnyMap mergedProperties;
+            int changeCount;
```

Comment:
`changeCount` should be `unsigned long`; its never going to be a value less than 0.

Reply:
fixed.